src/styles/variables.ts

```diff
@@ -0,0 +1,83 @@
+/* File auto-generated */
```

Comment:
Is this the strictest possible constant definition in TS? In flow you can also do this to make it a constant string instead of regular string:

```
const black: '#000' = '#000'
```

Reply:
Good question, I can make it generate that. Ideally this should be an enum like this:

enum bdlVars {
    white = 'white',
    black = 'black',
    ...
}